modules/food-menu/food-item-detail.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+
+import * as React from 'react'
+import {Text, ScrollView, StyleSheet} from 'react-native'
+import {Column, Row} from '@frogpond/layout'
+import {ListRow, ListSeparator, Detail, Title} from '@frogpond/lists'
+import * as c from '@frogpond/colors'
+import map from 'lodash/map'
+import {DietaryTagsDetail} from './dietary-tags-detail'
+import {calculateAmount} from './lib/calculate-amount'
+import {type TopLevelViewPropsType} from '../../source/views/types'
```

Comment:
This needs to be replaced by `import type {NavigationScreenProp} from 'react-navigation'`, as we shouldn't use anything from /source within /modules.

The `type Props` should then become `navigation: NavigationScreenProp<*> & {state: {params: {icons: MasterCorIconMapType, item: MenuItem}}}` (technically there's something we should pass instead of `*` there, but I don't recall what it is, and the alternative should work and just be a bit more verbose.)